core/src/multi_proof.rs

```diff
@@ -8,26 +8,30 @@ use crate::{
 };
 use bitvec::{order::Msb0, view::BitView};
 
-// Each terminal node may have a set of siblings that are uniquely
-// used during the verification with its terminal node.
-// This struct contains the terminal node itself,
-// the depth at which the siblings start to be uniquely associated to that terminal,
-// and the siblings themselves.
+/// Each terminal node may have a unique set of sibling nodes
+/// that are exclusively used in its verification process.
```

Comment:
I found it a bit clearer before.

bikeshed: "Each terminal node may have a unique set of sibling nodes which are used only along its path".